### PR TITLE
fix(manifests): Multi-User manifests fixes for 1.3

### PR DESCRIPTION
**Pull request by yanniszark**
**Description of your changes:**

Fixes https://github.com/kubeflow/pipelines/issues/5414
Fixes https://github.com/kubeflow/pipelines/issues/5415

### CODE REVIEW

**Comment by zijianjoy** [comment]
/lgtm
/approve

Thank you Yannis for the fix!

File: manifests/kustomize/base/installs/multi-user/istio-authorization-config.yaml [hunk 1]
```diff
@@ -12,18 +12,36 @@ spec:
...
+  rules:
+  - from:
+    - source:
+        principals:
+        - cluster.local/ns/kubeflow/sa/ml-pipeline
+        - cluster.local/ns/kubeflow/sa/ml-pipeline-ui
+        - cluster.local/ns/kubeflow/sa/ml-pipeline-persistenceagent
+        - cluster.local/ns/kubeflow/sa/ml-pipeline-scheduledworkflow
+        - cluster.local/ns/kubeflow/sa/ml-pipeline-viewer-crd-service-account
+        - cluster.local/ns/kubeflow/sa/kubeflow-pipelines-cache
```
Comment (Bobgy) [inline manifests/kustomize/base/installs/multi-user/istio-authorization-config.yaml:34]: nit: I think some entries are not necessary, but we can prune them after merging this PR

**Review by Bobgy — COMMENTED** [review]
Thank you!
/LGTM
/approve

**Pull request merged**